sources/packages/web/src/components/aest/students/modals/ApproveApplicationOfferingChangeRequestModal.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <modal-dialog-base :showDialog="showDialog" title="StudentAid BC Declaration">
+    <template #content>
+      <p>
+        Please review and accept the StudentAid BC declaration to allow the
+        institution to make proposed changes to your application (on your
+        behalf).
+      </p>
+      <content-group-info class="my-3">
+        <v-row
```

Comment:
A bit confused, why do you went for row col approach here?

Reply:
There are a lot many places in code where we are using the row col approach inside `content-group`. For instance, `OfferingView` and `SearchStudents` components and many others.

In the screenshot below, `v-row, v-col` results in the addition of `divs` for block level separation.

<img width="1913" alt="image" src="https://github.com/bcgov/SIMS/assets/7859295/54258f98-52ca-4a05-bab7-1a1504d5cd78">

Reply:
@dheepak-aot @andrepestana-aot @guru-aot suggestions?

Reply:
Again for the benefit of time, can connect with reviewer and either fix it or check if it is not a blocker.

Reply:
@sh16011993 pls take a look when to use `row` `col`  approach, https://vuetifyjs.com/en/components/grids/